app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -56,29 +47,22 @@ jest.mock('@/api/apiFunctions', () => ({
       week: 1,
     }),
   ),
-  getCurrentUserEntries: jest.fn(() =>
-    Promise.resolve([
-      {
-        id: '123',
-        week: 1,
-        selectedTeams: [],
-      },
-    ]),
-  ),
```

Comment:
Is the Promise.resolve no longer necessary here, which is why you prompted to delete it in favor for just a `jest.fn()`?

Reply:
correct because I mock the response differently per test.  leaving it here it would always call the empty array of teams which didn't work for some tests.